lib/feature/browser/browser_tab_view/browser_tab_view.dart

```diff
@@ -5,6 +5,7 @@ import 'package:app/data/models/models.dart';
 import 'package:app/di/di.dart';
 import 'package:app/feature/browser/browser.dart';
 import 'package:app/feature/browser/browser_tab_view/browser_error_view.dart';
+import 'package:app/feature/browser/browser_tab_view/browser_view_events_listener/browser_view_events_listener_cubit.dart';
```

Comment:
Should we create barrel file there?

Reply:
`browser_tab_view.dart` is a barrel file itself, so i'm not sure we need create another one